extensions/jwks/README.md

```diff
@@ -0,0 +1,34 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-extensions">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector Extension:<br />Last Commit Info</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=bug_report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=feature_request.md">Request Feature</a>
+  </p>
+</div>
+
+## About this Extension
+The JWKS-Extension provides an endpoint in the default API of the EDC-Connector, that returns the JWKS of the connector.
+It can be accessed using the `:{WEB_HTTP_PORT}/{WEB_HTTP_PATH}/jwks` (default: `:11001/api/jwks`) endpoint.
+
+## Why does this extension exist?
+The JWKS-endpoint can be used to validate tokens issued by the EDC-Connector. This can be particular useful for the DAPS.
```

Comment:
Both these documentations are correct, yet still technical. The goal is to give an introduction for wandering souls stumbling over this directory

"This is part of our goal of simplifying the on-boarding process of connectors to a DAPS. With this the connector can self-declare its public certificate."